CHANGELOG.md

```diff
@@ -25,6 +25,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ---
 
+## [0.28.0] - 2024-10-29
+
+### Added
+
+- Add DisableSSHCAUser and DisableSSHCAHost options to linkedca GCP provisioner (smallstep/certificates#2045)
+- Use dnsNamesSubsetValidator for IID provisioners (smallstep/certificates#2044)
```

Comment:
You can be more precise. The first allows you to add those properties using the `step ca provisioner add|update` commands. The second allows you to use only a subset of SANs instead of all of them.